CHANGELOG.md

```diff
@@ -1,22 +1,32 @@
 ## 0.0.1
+
 Package has 3 basic functionalities on android and IOS via method channel.
     - Disable screenshot support in app
     - Enable screenshot support in app
     - Toggle between enable and disable state
 
-## 0.0.1+1 
-Updated readme and added sample usage. 
+## 0.0.1+1
+
+Updated readme and added sample usage.
+
+## 0.0.1+2
 
-## 0.0.1+2 
 - Adopted MIT license
 
-## 0.0.1+3 
-- Reverted to BSD 3 license 
+## 0.0.1+3
+
+- Reverted to BSD 3 license
 - Added documentation
 - Made `NoScreenshot` class a singleton
 
-## 0.0.1+4 
-- Fixed issue #1[Crashes app when backgrounded on iOS](https://github.com/FlutterPlaza/no_screenshot/issues/1) 
+## 0.0.1+4
+
+- Fixed issue #1[Crashes app when backgrounded on iOS](https://github.com/FlutterPlaza/no_screenshot/issues/1)
+
+## 0.0.1+5
+
+- Fixed broken link from pub dev analyses
+
+## 0.0.1+6
 
-## 0.0.1+5 
-- Fixed broken link from pub dev analyses 
\ No newline at end of file
+- Fixed Non Implemented function in android life-cycle
```

Comment:
Non Implemented should be in lowercase if it's not a class name or  unique component you wish to highlight